crates/python/qcs_sdk/api.pyi

```diff
@@ -337,31 +360,22 @@ async def get_quilc_version(
     """
     ...
 
-
-async def get_quilc_version(
```

Comment:
What happened to `get_quilc_version`? Looks like it got deleted.

Reply:
There was actually 2 duplicate definitions, so I removed the redundant one